metricflow/plan_conversion/select_column_gen.py

```diff
@@ -33,11 +33,11 @@ def as_tuple(self) -> Tuple[SqlSelectColumn, ...]:
         """Return all select columns as a tuple."""
         return tuple(
             # Preferred order is to have metrics first.
-            self.metric_columns
+            self.identifier_columns
             + self.measure_columns
             + self.dimension_columns
             + self.time_dimension_columns
```

Comment:
I just noticed the time dimension and measure here also. per @belindabennett's suggestions, can we order these by:
time dimension, 
identifiers, 
dimensions, 
metrics, 
measures